codalab/worker/docker_utils.py

```diff
@@ -261,6 +265,39 @@ def get_container_stats(container):
     return stats
 
 
+@wrap_exception('Unable to check Docker API for container')
+def get_container_stats_on_mac(container: docker.models.containers.Container):
+    if container_exists(container):
+        return get_container_stats_helper_by_name(container.name)
+    else:
+        return '', ''
+
+
+@wrap_exception('Unable to check Docker API for container')
+def get_container_stats_helper_by_name(container_name: str) -> Tuple[str, str]:
+    try:
+        container_stats: dict = client.containers.get(container_name).stats(stream=False)
+        cpu_used: int = int(container_stats['cpu_stats']['cpu_usage']['total_usage'])
+        total_cpu: int = int(container_stats['cpu_stats']['system_cpu_usage'])
+        cpu_usage: str = str(round(float(cpu_used / total_cpu) * 100, 3)) + " %"
+        memory_usage: str = (
```

Comment:
It might be cleaner to use string format. Can you also add a comment of an example before and after the formatting and rounding?